site/SiteConstants.ts

```diff
@@ -0,0 +1,13 @@
+// See https://polyfill.io/v3/url-builder/ for a list of all supported features
+const polyfillFeatures = [
+    "es2019", // Array.flat, Array.flatMap, Object.fromEntries, ...
+    "es2020", // String.matchAll, Promise.allSettled, ...
+    "es2021", // String.replaceAll, Promise.any, ...
+    "es2022", // Array.at, String.at, ...
```

Comment:
We don't currently use these, but I found it nice to include them already.
Most users are using modern browsers, where the polyfill will come back empty anyways. So there's not much of an overhead.